src/Components/InputCard/InputCard.jsx

```diff
@@ -48,18 +82,13 @@ export const InputCard = ({ setCreateNewCard, edit }) => {
           }}
           autoFocus
         />
-        <textarea
-          type="text"
-          className="text-new-note pdt-0 mt-0"
-          placeholder="Take a note..."
-          rows="2"
-          value={inputCardDetails.description}
-          onChange={(e) => {
-            setInputCardDetails({
-              ...inputCardDetails,
-              description: e.target.value,
-            });
-          }}
+        <ReactQuill
+          theme="snow"
+          value={body}
+          onChange={setBody}
+          placeholder={"Take a note..."}
+          modules={modules}
+          formats={formats}
```

Comment:
UI looks neat and code looks good, it is good to see rich text editor for notes, yo can proceed to merge